include/pingcap/kv/RegionClient.h

```diff
@@ -83,6 +88,76 @@ struct RegionClient
         }
     }
 
+    template <typename RESP>
+    struct StreamRequestCtx
```

Comment:
Seems that we can provide convenient methods like read and finish over this structure, so that the caller doesn't need to take care of "first response".